ceph_deploy/hosts/fedora/__init__.py

```diff
@@ -18,7 +18,7 @@ def choose_init():
 
     Returns the name of a init system (upstart, sysvinit ...).
     """
-    return 'sysvinit'
+    return 'systemd'
```

Comment:
I'm less worried about fc21...

The bigger problem is that with these changes you can no longer install hammer, which is the currently recommended stable release.  We need to figure out how to make the init system choice dependent on the Ceph version installed...